refl1d/probe.py

```diff
@@ -1712,7 +1750,8 @@ def shared_beam(self, intensity=1, background=0,
         back_absorption = Parameter.default(back_absorption,
                                             name="back_absorption",
                                             limits=[0, 1])
-        theta_offset = Parameter.default(theta_offset, name="theta_offset")
+        theta_offset = Parameter.default(theta_offset,
+                                            name="theta_offset")
```

Comment:
Optional: I'm leaning toward black's strategy of breaking after the parenthesis for long argument lists. This leads to a consistent four character indent on every line rather than trying to line up the indent with the parenthesis.
So:
```
theta_offset = Parameter.default(
    theta_offset, name="theta_offset")
```
Black may be more verbose about it for consistency:
```
theta_offset = Parameter.default(
    theta_offset,
    name="theta_offset",
)
```
but I prefer to the more compact vertical form.